proposed/authelia/latest/questions.yml

```diff
@@ -0,0 +1,128 @@
+categories:
+- Authentication
+questions:
+- variable: authelia.enableHealtcheck
+  default: false
+  description: "Enable kubernetes health check"
+  label: Enable health check
+  type: boolean
```

Comment:
probably is better to add a group for those ungrouped variables.

Reply:
Ok let me think of something.